extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampCommentsInfoItemExtractor.java

```diff
@@ -86,7 +86,12 @@ public boolean isPinned() {
     }
 
     @Override
-    public boolean isUploaderVerified() throws ParsingException {
+    public boolean isUploaderVerified() {
         return false;
     }
+
+    @Override
+    public int getStreamPosition() {
+        return -1;
```

Comment:
Create a constant for this in `CommentsInfoItem`, something like `NO_STREAM_POSITION`

Reply:
Good idea :+1: 
I also added `NO_LIKE_COUNT` for comment likes